### PR TITLE
Changes to contribution guideline

### PR DESCRIPTION
**Pull request by rshym**
- Fix formatting of `reviewers.md`
- Use Github Flavored Markdown instead of kramdown
- Rework 'Review process' chapter
- Move contribution guideline from `CONTRIBUTING.md` to `contribution.md`

### CODE REVIEW

**Comment by rshym** [comment]
You need to create Pull Request on GitHub with your changes and ask appropriate persons to review your commits.

If you are not sure who this 'appropriate person' is, look into the list of reviewers https://github.com/xen-troops/xen-troops.github.io/blob/master/reviewers.md.

After comments are addressed, reviewers need to explicitly say "this patch is okay" by providing a `Reviewed-by` or `Acked-by` tag.

**Comment by arminn** [comment]
Reviewed-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ The explanation body will be committed to the permanent source changelog, so sho
 As in the summary phrase , it is important to be both succinct as well as descriptive.
 
 #### Review process
-You must create Pull Request on github with your changes and ask appropriate persons to review the commits you made. After all comments addressed, reviewers needs to explicitly say "this patch is okay" by providing Reviewed-by tag.
+You must create Pull Request on github with your changes and ask appropriate persons to review the commits you made. If you are not sure who this 'appropriate person' is, look into the list of reviewers https://github.com/xen-troops/xen-troops.github.io/blob/master/reviewers.md. After all comments addressed, reviewers needs to explicitly say "this patch is okay" by providing Reviewed-by tag.
```
Comment (lorc) [inline CONTRIBUTING.md]: I wanted to ask, why this link does not point to https://xen-troops.github.io/reviewers

But then I saw.... I think we need to fix that page and put link to it there.

Comment (rshym) [reply CONTRIBUTING.md]: Yes, table looks terrible.
I''m not sure, but seems we need to choose the proper markdown processor in `_config.yml` 
https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/setting-a-markdown-processor-for-your-github-pages-site-using-jekyll

Looks like `GFM` supports tables and `kramdown` support not.

Need additional investigation.

Comment (lorc) [reply CONTRIBUTING.md]: Okay, then we need to switch to `GFM`. I don't see, why not

**Comment by rshym** [comment]
PR is reworked and ready for re-review.
The final result can be checked at https://rshym.github.io/

**Comment by lorc** [comment]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by rshym** [comment]
Reviewers are added.
Ready to be merged.

**Pull request merged**